.travis.yml

```diff
@@ -35,36 +35,32 @@ before_cache:
   - rm -rf $HOME/.gradle/caches/*/workerMain/
   - rm -rf $HOME/.gradle/caches/*/gc.properties
   - rm -rf $HOME/.gradle/caches/journal-1/
-  - rm -rf node_modules/.cache/uglifyjs-webpack-plugin/
-  - rm -rf node_modules/.cache/terser-webpack-plugin/
 
 cache:
   directories:
     - $HOME/.gradle/caches/
     - $HOME/.gradle/wrapper/
-    - node_modules
 
 jobs:
   include:
     - stage: "Prepare Dependencies"
       name: "Prepare Dependencies"
       script:
         - ./gradlew downloadDependencies downloadTestDependencies downloadLinters
-        - npm install
-        - npm update
+        - npm ci
```

Comment:
You can remove this line entirely, because this stage is meant to be caching dependencies.